lib/remote/jsonrpcconnection-pki.cpp

```diff
@@ -66,8 +66,14 @@ Value RequestCertificateHandler(const MessageOrigin::Ptr& origin, const Dictiona
 			}
 			logmsg << " signed by our CA.";
 		} catch (const std::exception &ex) {
-			logmsg << " not signed by our CA";
-			if (const unsigned long *openssl_code = boost::get_error_info<errinfo_openssl_error>(ex)) {
+			const unsigned long *openssl_code = boost::get_error_info<errinfo_openssl_error>(ex);
+			if (*openssl_code == X509_V_ERR_CERT_REVOKED) {
```

Comment:
Are you sure that the pointer `openssl_code` is not NULL and you can just dereference it?